docs/pages/project/contributing/contributing-error.md

```diff
@@ -9,27 +9,127 @@ category: contributing
 ---
 
 Meshery pervasively uses MeshKit as a golang and service mesh management-specific library in all of its components. MeshKit helps populate error messages with a uniform and useful set of informative attributes. 
+
+This tool analyzes, verifies and updates error codes in Meshery source code trees. It extracts error details into a file that can be used for publishing all error code references on the Meshery website. The objective to create this was to avoid centralized handling of error codes, automating everything
+
+Inorder to create a meshery error object, you will need to create a custom wrapper object for the native golang error. This can be done from the <a href="https://github.com/layer5io/meshkit/tree/master/errors">Meshkit Error</a> package. 
+
+Use the `errors.New()` function to create a new instance of the error object and pass situation-specific attributes as function arguments. 
 The below are the listed attributes:
 - Code
 - Short Description
 - Long Description
 - Probable Cause
 - Suggested Remediation
 
-Inorder to create a meshery error object, you will need to create a custom wrapper object for the native golang error. This can be done from the <a href="https://github.com/layer5io/meshkit/tree/master/errors">Meshkit Error</a> package. 
+### Syntax
+     errors.New(ErrExampleCode, errors.Alert, []string{"<short-description>"}, []string{"<long-description>"}, []string{"<probable-cause>"}, []string{"<suggested remediation>"})
+
 
-Use the `errors.New()` function to create a new instance of the error object and pass situation-specific attributes as function arguments. See the following example for reference.
+It is intended to be run locally and as part of a CI workflow.
+
+- Errors names and codes are namespaced to components, i.e. they need to be unique within a component, which is verified by this tool.
```

Comment:
These are some nice bullet points :white_check_mark:. But they seem to highlight more about the utility. I think so much info might confuse contributors.
Can these be made more like: "some rules for making error codes" and then mentioning some conventions to follow that abide by the utility?

Reply:
`Errors names and codes` to `Errors name and code` might be?